src/buckets/components/context/csvUploaderProvider.tsx

```diff
@@ -138,9 +164,22 @@ export const CsvUploaderProvider: FC<Props> = React.memo(({children}) => {
             }
             time = table.getColumn('_time')?.[i] ?? Date.now()
             value = table.getColumn('_value')?.[i] ?? field ?? ''
+            // Adds quotes to values if _value is of string type
+            // https://docs.influxdata.com/influxdb/cloud/reference/syntax/line-protocol/#quotes
+            value =
+              table.getColumnType('_value') === 'string' && value
+                ? '"' + value + '"'
```

Comment:
To this point, we can probably just use template strings

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals